spartan/aztec-network/templates/validator.yaml

```diff
@@ -113,6 +113,8 @@ spec:
               value: "{{ .Values.validator.p2p.enabled }}"
             - name: VALIDATOR_DISABLED
               value: "{{ .Values.validator.validator.disabled }}"
+            - name: VALIDATOR_RE_EX
```

Comment:
Nit, should it be `reexecute`? instead of the shorthand, just that seeing stuff like `reEx` later is not a much used shorthand so might make sense to make it more understandable in line with some of the others 🤷